build/helper.py

```diff
@@ -569,12 +569,20 @@ def get_function_rst(fname, config, indent=0):
         rst += '\n\n' + (' ' * indent) + ':rtype: tuple ('+ ', '.join([p['python_name'] for p in output_params]) + ')\n\n'
         rst += (' ' * (indent + 4)) + 'WHERE\n'
         for p in output_params:
-            rst += '\n' + (' ' * (indent + 4)) + '{0} ({1}): '.format(p['python_name'], p['python_type']) + '\n'
+            p_type = p['intrinsic_type']
```

Comment:
Something to consider, this function is getting pretty long and hard to follow. Also, docstring is outdated. It could use a bit of comments or even refactoring / breaking up in smaller functions.

At some point we'll want to break helper.py up into logical pieces too.